cmd/fips/main.go

```diff
@@ -0,0 +1,234 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"go/ast"
+	"log"
+	"os"
+	"strings"
+
+	"golang.org/x/tools/go/packages"
+)
+
+const description = `
+Fips examines Go source code and reports which public APIs are backed by openssl
+primitives, together with other useful information such as the
+openssl function names.
+
+Set the appropriate GOROOT when checking std packages not in your default Go toolchain. 
+
+Packages are specified using the notation of "go list".
+This command checks the package in the current directory:
+
+	go run ./cmd/fips
+
+whereas this one checks the std crypto package in GOROOT, as specified:
+
+	go run ./cmd/fips crypto/...
```

Comment:
I think adding an example using `-goos` would be useful, to show why the option exists. 😄

```
The GOOS can also be specified. This allows, for example, examining the openssl APIs while running Windows:

	go run ./cmd/fips crypto/... -goos linux
```

Reply:
done!